.env.example

```
@@ -0,0 +1,25 @@
+NODE_ENV=development
+PORT=5742
+DATABASE_URL=postgres://app_user:change-me@gimbal.fobdongle.com:5432/aformulationoftruth?sslmode=require
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the NODE_ENV key

(UnorderedKey)